### PR TITLE
[libtorrent] Update to 1.2.11

### PR DESCRIPTION
**Pull request by FranciscoPombal**
- Remove python bindings patch, upstream now has the relevant fixes
- Remove patch related to finding boost libs, not needed anymore
- The [python] and [test] features no longer depend on [deprfun]

---

Previously: https://github.com/microsoft/vcpkg/pull/13341 and https://github.com/microsoft/vcpkg/pull/13375

The `[python]` feature is still a no-go on Windows. All else works fine on my end.

Updating to 2.0 will take a while, there is ongoing discussion about that in https://github.com/microsoft/vcpkg/pull/14293. In the meantime, this one can be merged to provide the latest fixes.

### CODE REVIEW

**Review by glassez — APPROVED** [review]

**Review by zywo — APPROVED** [review]

File: ports/libtorrent/CONTROL [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Source: libtorrent
-Version: 1.2.10
+Version: 1.2.11
 Port-Version: 2
```
Comment (NancyLi1013) [inline ports/libtorrent/CONTROL:3]:
```suggestion
```
This field is no longer needed for new version update.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/libtorrent/CONTROL (hunk 1 above)
Comment (FranciscoPombal) [reply ports/libtorrent/CONTROL]: fixed.

**Comment by FranciscoPombal** [comment]
@NancyLi1013 Sorry for the delay. All comments addressed.

**Comment by NancyLi1013** [comment]
Thanks for your update. Have you tested these features? If not, I will test them on my local.

**Comment by FranciscoPombal** [comment]
@NancyLi1013 

> Thanks for your update. Have you tested these features? If not, I will test them on my local.

Yes, on both Windows and Linux (Ubuntu 18.04). But feel free to reproduce on your local setup.

**Review by NancyLi1013 — APPROVED** [review]

**Review by glassez — APPROVED** [review]

**Comment by NancyLi1013** [comment]
LGTM now, thanks for your PR @FranciscoPombal .

**Comment by glassez** [comment]
Excuse my curiosity, is there something else that prevents this PR from being merged?

**Comment by BillyONeal** [comment]
Thanks for your help!

**Pull request merged**